schemas/common/threshold.cue

```diff
@@ -20,6 +20,7 @@ package common
 }
 
 #thresholds: {
+	mode?:          string
```

Comment:
Could be good to define the supported modes here (similar to [calculation.cue](https://github.com/perses/perses/blob/main/schemas/common/calculation.cue)), so something like: `mode?:  "Absolute" | "Percent"`